### PR TITLE
Objects definition

### PR DESCRIPTION
**Pull request by asadate**
Broke up the definitions.json file into multiple separate objects specifications files. Updated links to object references in the other files

### CODE REVIEW

File: part-spec/Registers.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "$id": "https://github.com/edatasheets/edatasheets.github.io/blob/main/part-spec/Registers.json",
-  "$schema": "http://json-schema.org/draft-07/schema#",
-  "definitions": {
+    "$id": "https://github.com/edatasheets/edatasheets.github.io/blob/main/part-spec/Registers.json",
```
Comment (rsnancollas) [inline part-spec/Registers.json]: I think we should keep this consistent and use registers.json not Registers.json.

File: part-spec/conditionalProperty.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "$id": "https://github.com/edatasheets/edatasheets.github.io/blob/main/part-spec/conditionalProperty.json",
```
Comment (rsnancollas) [inline part-spec/conditionalProperty.json:2]: would it make sense for this to be part of unit?

**Review by rsnancollas — APPROVED** [review]
This looks good.

One thing we can think about (and feel free to decline this if you think it's not worth the effort) is putting the files that were formally in the "definitions" file in a "definitions" folder so that it's a little easier to see where the parts are - or having a folder for each. If you don't think it's worth the effort though, we can leave as is. Thanks for making this change.

File: part-spec/Registers.json (hunk 1 above)
Comment (asadate) [reply part-spec/Registers.json]: changed

File: part-spec/conditionalProperty.json (hunk 2 above)
Comment (asadate) [reply part-spec/conditionalProperty.json:2]: Let's walk through an example to see how this would look like

**Pull request merged**